appveyor.yml

```
@@ -94,24 +96,28 @@ environment:
       PLATFORM: 'x64'
       CRYPTO_BACKEND: 'WinCNG'
       ENABLE_DEBUG_LOGGING: 'ON'
+      ENABLE_ECDSA_WINCNG: 'ON'
```

```suggestion
      CRYPTO_BACKEND: 'WinCNG'
      ENABLE_ECDSA_WINCNG: 'ON'
      ENABLE_DEBUG_LOGGING: 'ON'
```
Tiny nit, to keep the WinCNG stuff next to each other.